client/js/lounge.js

```diff
@@ -297,14 +297,14 @@ $(function() {
 		utils.setNick(sidebar.find(".chan.active").closest(".network").data("nick"));
 	}
 
-	$("#nick-value").keypress(function(e) {
+	$("#nick-value").on("keypress", function(e) {
```

Comment:
Might as well use arrow functions while you're at it, but nbd.

Reply:
Didn't want to break anything unintentionally.

Reply:
Argh yes, I always forget jQuery is funky when it comes to `this` reference.